usecases/Concrete/dodo.py

```diff
@@ -45,11 +45,21 @@ def task_emodul():
     }
     yield {
         'basename': 'extract emodul metadata',
-        'actions': ['python knowledgeGraph/emodul/emodul_metadata_extraction.py']
+        'actions': ['python knowledgeGraph/emodul/emodul_metadata_extraction.py'],
+        'targets': ['knowledgeGraph/emodul/E-modul-processed-data/emodul_metadata.csv']
+    }
+    yield {
+        'basename': 'calculate emodul',
+        'actions': ['python knowledgeGraph/emodul/emodul_calculation.py'],
```

Comment:
not sure why we calculate the E-modulus here already, that is IMO part of the processing of the data afterwards.